hnn_core/gui/gui.py

```diff
@@ -26,15 +26,307 @@
                              _read_legacy_params)
 from hnn_core.viz import plot_dipole
 
-THEMECOLOR = "#8A2BE2"
 
+class HNNGUI:
+    """HNN GUI class
+
+    Parameters
+    ----------
+    theme_color : str
+        The theme color of the whole dashboard.
+    log_window_height: str
+        The height of the log output window (in pixel).
+    visualization_window_width: str
+        The width of the visualization window (in pixel).
+    visualization_window_height: str
+        The height of the visualization window (in pixel).
+    left_sidebar_width: str
+        The width of the left side bar (in pixel).
+    TODO: add more parameters.
+
+    Attributes
+    ----------
+    params: dict
+        The parameters to use for constructing the network.
+    TODO: add more attributes
```

Comment:
Add them all! Otherwise it's hard to evaluate if the class needs restructuring or not